internal/cmd/doctorcmd.go

```diff
@@ -187,7 +187,7 @@ func (c *Config) runDoctorCmd(cmd *cobra.Command, args []string) error {
 			name:        "age-command",
 			binaryname:  c.Age.Command,
 			versionArgs: []string{"-version"},
-			versionRx:   regexp.MustCompile(`v(\d+\.\d+\.\d+\S*)`),
+			versionRx:   regexp.MustCompile(`(v\d+\.\d+\.\d+\S*)|(\(devel\))`),
```

Comment:
This seems unrelated to this PR.

Reply:
Wow, I was playing with it and not even noticed it!